.github/workflows/main.yml

```diff
@@ -19,36 +19,30 @@ jobs:
       matrix:
         include:
           # MANDATORY CHECKS USING CURRENT DEVELOPMENT INTERPRETER
-          - python-version: 3.8.6
-            dependencies: >
-              dbus-python==1.2.16
-              pylint==2.4.4
-              psutil==5.6.7
+          - dependencies: >
+              dbus-python-devel
+              pylint
+              python3-psutil
+            image: fedora:33
             task: PYTHONPATH=./src make -f Makefile lint
-          - python-version: 3.8.6
-            dependencies: black==20.8b1 isort==4.3.21
+          - dependencies: >
+              black
+              python3-isort
+            image: fedora:28
```

Comment:
formatting + linting should also be current development fedora release, so this should be 32 also.